### ISSUE TITLE
Get build failure reasons

### ISSUE THREAD

**Issue by ASaiAnudeep** [microsoft/azure-devops-node-api#489]
I'm trying to get build failure reasons that are usually displayed in the build summary page - `https://dev.azure.com/{org}/{project}/_build/results?buildId={id}&view=results`

I found out that **getBuildReport** method will provide the build errors and warnings. But the default type of the content is HTML which is not friendly for a node.js app. I need it in a JSON format. From the [docs](https://docs.microsoft.com/en-us/rest/api/azure/devops/build/report/get?view=azure-devops-rest-6.0) the supported types are missing.

Is there a better way to get the build failure reasons?

**Comment by golubev-ivan** [comment]
Hi @ASaiAnudeep, I investigated your issue. The response type of this query is JSON, but it contains HTML data. As far as I see, there is no way to required data in just JSON form. If it required option for you, please submit an issue in Microsoft Developer Community to have the right eyes on this request, since it is required changes in Azure DevOps API to support such behavior.

**Comment by golubev-ivan** [comment]
@ASaiAnudeep Do you have any additional questions regarding azure-devops-node-api itself?

**Comment by ASaiAnudeep** [comment]
Nope. Thanks for the quick help. If you find a better API/method to get the build failure reasons, please update this thread.
